WebAnalyzerTests/Controllers/TestController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace WebAnalyzerTests.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class TestController : ControllerBase
+    {
+
+        private readonly MyDBContext _ctx;
+        public TestController(MyDBContext ctx)
+        {
+            _ctx = ctx;
+        }
+
+        [HttpGet]
+        public bool GetEntity(string name)
+        {
+            return entityExists(name);
+        }
+
+        private bool entityExists(string name)
```

Comment:
Change the name of this method to conform to naming conventions.
```suggestion
        private bool EntityExists(string name)
```